lib/account.php

```diff
@@ -69,18 +72,19 @@ public function getImapConnection() {
 			$host = $this->account->getInboundHost();
 			$user = $this->account->getInboundUser();
 			$password = $this->account->getInboundPassword();
+			$password = $this->crypto->decrypt($password);
```

Comment:
> Problem still exists - at this point the password gets decrypted although at this moment the password is unencrypted in the variable.

which is not a case we cover - the password has to be encrypted as taken from the db.